src/common/utils/dateFormatter.ts

```diff
@@ -0,0 +1,41 @@
+export const isBefore = (date: Date | string, dateToCompare: Date | string): boolean => {
+  if (typeof date === 'string') date = new Date(date);
+  if (typeof dateToCompare === 'string') dateToCompare = new Date(dateToCompare);
+  return date.getTime() < dateToCompare.getTime();
+};
+
+export const isAfter = (date: Date | string, dateToCompare: Date | string): boolean => {
+  if (typeof date === 'string') date = new Date(date);
+  if (typeof dateToCompare === 'string') dateToCompare = new Date(dateToCompare);
+  return date.getTime() > dateToCompare.getTime();
+};
+
+export const differenceInSeconds = (laterDate: Date | string, earlierDate: Date | string): number => {
+  if (typeof laterDate === 'string') laterDate = new Date(laterDate);
+  if (typeof earlierDate === 'string') earlierDate = new Date(earlierDate);
+  return Math.floor((laterDate.getTime() - earlierDate.getTime()) / 1000);
+};
+
+export const subMinutes = (date: Date | string, amount: number): Date => {
+  if (typeof date === 'string') date = new Date(date);
+  const newDate = new Date();
+  newDate.setTime(date.getTime() - amount * 60 * 1000);
+  return newDate;
+};
```

Comment:
```suggestion
export const subMinutes = (date: Date | string, amount: number): Date => {
  date = toDate(date);
  date.setMinutes(date.getMinutes() - amount);

  return date;
};
```
바로 minute 바꿔줘도 될 거 같아요~